docker-compose/README.md

```diff
@@ -0,0 +1,54 @@
+## Overview
+
+This is the docker-compose setup to run 
+
+- **mimoto-service** which act as BFF for Inji mobile and backend for Inji web.
+- **inji-web** and **inji-web-proxy** for frontend
+
+This is not for production use.
+
+## Building the inji-web-proxy image locally.
+
+> cd inji-web-proxy && docker build -t inji-web-proxy:local .
+
+## Building the inji-web image locally.
+
+> cd ../inji-web && docker build -f Dockerfile.local -t inji-web:local .
+
+## What is in the docker-compose folder?
+
+1. certs folder holds the p12 file which is being created as part of OIDC client onboarding.
+2. "config" folder holds the mimoto system properties file, issuer configuration and credential template.
+3. "docker-compose.yml" file with mimoto setup.
+
+## How to run this setup?
+
+1. Add Id providers as an issuer in mimoto-issuers-config.json
+
+2. Add verifiers clientId and redirect Uri in mimoto-trusted-verifiers.json for Online Sharing
+
+3. Start esignet services and update esignet host references in mimoto-default.properties and mimoto-issuers-config.json
```

Comment:
mention start esignet(authorisation server) or used existing esignet deployed on cloud

Reply:
Updated the readme